src/ddmd/dclass.d

```diff
@@ -179,6 +179,26 @@ struct ClassFlags
     alias hasDtor = Enum.hasDtor;
 }
 
+/**
+ * CFM stands for Class Flag Masks. Each ClassDeclaration AST node
+ * allocates a bitfield which stores various class properties.
+ * These properties can be retrieved or updated by using
+ * these masks.
+ */
+enum CFM : ubyte
```

Comment:
I highly recommend calling it `ClassFlagMasks` instead of `CFM`. That will make everything more clear and avoid the extra explanation in the documentation.

Reply:
Or just `ClassFlags`